src/uct/ib/mlx5/dc/dc_mlx5_ep.h

```diff
@@ -389,6 +389,10 @@ uct_dc_mlx5_dci_pool_init_dci(uct_dc_mlx5_iface_t *iface, uint8_t pool_index,
     }
 
     return UCS_OK;
+
+err:
+    uct_dc_mlx5_destroy_dci(iface, dci_index);
```

Comment:
err_destroy:
uct_dc_mlx5_destroy_dci(iface, dci_index);
err:
return status;

and goto err from line 372